<launch>
    <rosparam command="load" file="$(find movement_controller)/param/pid_param.yaml"/>
    <rosparam command="load" file="$(find movement_controller)/param/waypoint_param.yaml"/>

    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch"/>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find movement_controller)/rviz/config.rviz"/>

    <node name="control_pid" pkg="movement_controller" type="control_pid" output="screen"/>
</launch>